public/styles/timeline.css

```diff
@@ -1,10 +1,3 @@
-@font-face {
-  font-family: "Edo SZ";
-  src: url("../fonts/edosz.ttf");
-  font-display: block;
-  unicode-range: U+0-22, U+24-29, U+2B-3F, U+41-D6, U+D8-24F;
```

Comment:
This `unicode-range` skips the characters that aren't actually available in this font. Since you've moved it to `base.css` then this should get included there too.

Reply:
Added